prow/oss/cluster/crier-ghapp.yaml

```diff
@@ -38,6 +38,7 @@ spec:
         - --github-enabled-org=chaotoppicks
         - --github-enabled-org=looker
         - --github-enabled-repo=GoogleCloudPlatform/oss-test-infra
+        - --github-enabled-repo=GoogleCloudPlatform/testgrid
```

Comment:
Don't we also need to disable on the bot based `hook` and `crier` instances?

Reply:
good catch

Reply:
fixed